clients/drcachesim/drpt2trace/pt2ir.h

```diff
@@ -293,6 +304,9 @@ class pt2ir_t {
     /* The libipt image section cache. */
     struct pt_image_section_cache *pt_iscache_;
 
+    /* The libipt elf file image. */
+    struct pt_image *pt_image_;
```

Comment:
Maybe make the identifier a bit more descriptive. it is currently (almost) same as the struct.

Reply:
Remove it.